debug_empty_diff.py

```
@@ -0,0 +1,103 @@
+    config = Config()
+    github = GitHubClient(config, config.REPOSITORY_OWNER, config.REPOSITORY_NAME)
+    
+    pr_number = 92
+    commit_sha = "b6965e4"
+    
+    print("="*80)
+    print(f"🔍 Debugging PR #{pr_number} - Commit {commit_sha}")
+    print("="*80)
+    
+    # Get PR details
+    print(f"\n📋 Fetching PR details...")
```

Ruff (0.14.7)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)